web/packages/teleport/src/lib/tdp/codec.ts

```diff
@@ -523,19 +529,38 @@ export default class Codec {
     return buffer;
   }
 
+  static keyboardButtonMessageLength =
+    1 /* MessageType */ + 4 /* scanCode */ + 1 /* state */;
+
   // encodeKeyboardInput encodes a keyboard action.
   // Returns null if an unsupported code is passed.
   // | message type (5) | key_code uint32 | state byte |
```

Comment:
This is no longer accurate, right?

Which would mean we need to update the RFD and the backend as well. (And probably use a new message ID)